apps/storybook/src/stories/main/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,198 @@
+import { useState } from 'react';
+import {
+  Copyright,
+  darkTheme,
+  globalStyles,
+  Logout,
+  LogoutConfirmation,
+  LogoutConfirmationMini,
+  SidebarFooter,
+  SidebarFooterMini,
+  SidebarHeader,
+  SidebarLink,
+  SidebarLinkList,
+  SidebarLinkMini,
+  SidebarMini,
+  SidebarNav,
+  SidebarProfile,
+  TermsAndConditions,
+} from './Sidebar.styled';
+
+interface INavLink {
+  name: string;
+  icon: string;
```

Comment:
Could add in an `onClick` function field here to pass in and use in the SidebarLink